public/app/features/provisioning/utils/getFormErrors.ts

```diff
@@ -61,3 +64,45 @@ export const getFormErrors = (errors: ErrorDetails[]): FormErrorTuple => {
 
   return [null, null];
 };
+
+/**
+ * Maps API error details to form error fields for ConfigForm
+ *
+ * @param errors Array of error details from the API response
+ * @returns Tuple with form field path and error message
+ */
+export type ConfigFormPath = Path<RepositoryFormData>;
+export type ConfigFormErrorTuple = [ConfigFormPath | null, { message: string } | null];
+export const getConfigFormErrors = (errors?: ErrorDetails[]): ConfigFormErrorTuple => {
```

Comment:
The `getFormErrors` has largely the same logic, except with a `repository.` prefix, could we reuse it?